File: dayEight/python/question_one.py
def quantitativeAptitude(numbers):
#    rearranged =  [len(numbers)]
    result = 0
    counter = 0
   
    
    
    for num in numbers:
        if num % 2 == 0:
            numbers =  num * 2 
#            for _ in rearranged:
#                _ = result
#                counter +=1
            
        
        
        else:
            numbers = num + 1
#            for _ in rearranged:
#                _ = result
#                counter +=1
           
       
#   return rearranged
    return numbers
    

numbers = [2,8,14,5,1]
quantitativeAptitude(numbers)
print(quantitativeAptitude)

File: dayEight/javaScript/questionOne.js
function quantitativeAptitude(numbers){
    let[] rearranged =  new [numbers.length];
    let result = 0;
    let counter = 0;
    
    for(let count = 0; count < numbers.length; count++){
        if(numbers[count] % 2 == 0){
           result =  numbers[count] * 2; 
           rearranged[counter] = result;
           counter++;
        }
        
        else{
            result = numbers[count] + 1;
            rearranged[counter] = result;
            counter++;
             }
 
        
        }
    
   return rearranged;
    }
    

let numbers = [2,8,14,5,1];
quantitativeAptitude(numbers);
